The Troubleshooters/sheet.json

```
@@ -0,0 +1,9 @@
+{
+	"html": "TheTroubleshooters.html",
+	"css": "TheTroubleshooters.css",
+	"authors": "Richard M.",
+	"roll20userid": "3094711",
+	"preview": "TheTroubleshooters.png",
+	"instructions": "RM June 2021 - This sheet is for the Swedish RPG The Troubleshooters (2021) published by Helmgast (https://helmgast.se/en/troubleshooters/). The coat of arms image from the passport cover and The Troubleshooters name and logo are (c) Helmgast and are used here with their kind permission. \n - When learning languages or developing attributes, use the checkboxes to record experience spent. \n - Condition descriptions are revealed by checking the boxes. \n When adding weapons on the Combat sheet, selecting the applicable skill will reveal the relevant roll buttons. \n This sheet can be edited and improved as long as the copyright statement is retained/updated for any relevant imagery or other content used.",
```

```suggestion
	"instructions": "RM June 2021 - This sheet is for the Swedish RPG [The Troubleshooters](https://helmgast.se/en/troubleshooters/) (2021) published by Helmgast. The coat of arms image from the passport cover and The Troubleshooters name and logo are (c) Helmgast and are used here with their kind permission. \n - When learning languages or developing attributes, use the checkboxes to record experience spent. \n - Condition descriptions are revealed by checking the boxes. \n When adding weapons on the Combat sheet, selecting the applicable skill will reveal the relevant roll buttons. \n This sheet can be edited and improved as long as the copyright statement is retained/updated for any relevant imagery or other content used.",
```
You can use markdown to make hyperlinks and to do some formatting/styling for the description.

I recommend looking into CSE, as there are new features it enables, and actually makes sheet creation simpler, as you don't need `sheet-` in from of every class. [Datalists](https://wiki.roll20.net/CSE#Datalist) is among the simplest features it enables, giving you the best of a text input & a select drop-down.